tools/dump_kubernetes.sh

```diff
@@ -46,6 +46,10 @@ usage() {
   error '  -m, --max-bytes          max total bytes, 0=no limit, default='${DEFAULT_MAX_LOG_BYTES}
   error '  -l, --label              if set, dump logs only for pods with given labels e.g. "-l app=pilot -l istio=galley"'
   error '  -n, --namespace          if set, dump logs only for pods in the given namespaces e.g. "-n default -n istio-system"'
+  error '  -p, --pod                if set, dump logs only for pod matching these values e.g. "-n istio-system -p istio-policy-7857db5f54-wj59d -p istio-pilot-5cb9d848b5-xx25q"'
```

Comment:
When I ran this it also got a bunch of other info. So is this just limiting the info returned, but there is still some basic info dumped as well?

Reply:
Yes this change addressed the initial use case from mandar, which was to filter by pods, I've basically filtered on pod name in the same codepath than before.